multi-node/config/wazuh_cluster/wazuh_worker.conf

```diff
@@ -222,8 +222,8 @@
   <global>
     <white_list>127.0.0.1</white_list>
     <white_list>^localhost.localdomain$</white_list>
-    <white_list>4.4.0.1</white_list>
-    <white_list>4.4.0.2</white_list>
+    <white_list>4.5.0.1</white_list>
+    <white_list>4.5.0.2</white_list>
```

Comment:
Why?

Reply:
It was an automatic modification, it was rolled back